test/gtest/uct/ib/test_dc.cc

```diff
@@ -440,6 +440,32 @@ UCS_TEST_P(test_dc, rand_dci_pending_purge) {
     flush();
 }
 
+UCS_TEST_P(test_dc, pool_indices)
+{
+    constexpr uint8_t max_dci_pools       = UCT_DC_MLX5_IFACE_MAX_DCI_POOLS;
```

Comment:
maybe define (or static assert) UCT_DC_MLX5_IFACE_MAX_DCI_POOLS according to UCT_DC_MLX5_EP_FLAG_POOL_INDEX_MASK?